src/main/java/com/avispl/dal/communicator/polycom/videoos/PolycomVideoOS.java

```diff
@@ -0,0 +1,935 @@
+                    getJsonProperty(jsonNode, "type").asText());
+            statistics.put(String.format("Microphones#Microphone %s HW Version", getJsonProperty(jsonNode, "number").asText()),
+                    getJsonProperty(jsonNode, "hwVersion").asText());
+            statistics.put(String.format("Microphones#Microphone %s SW Version", getJsonProperty(jsonNode, "number").asText()),
+                    getJsonProperty(jsonNode, "swVersion").asText());
+            statistics.put(String.format("Microphones#Microphone %s Mute", getJsonProperty(jsonNode, "number").asText()),
+                    String.valueOf("0".equals(getJsonProperty(jsonNode, "mute").asText())));
+        });
+    }
+
+    /**
+     * Retrieve device content sharing status
+     *
+     * @param statistics map to put values to
+     * @throws Exception during http communication
+     */
+    public void getContentStatus(Map<String, String> statistics) throws Exception {
+        String response = doGet(buildHttpUrl(CONTENT_STATUS), String.class);
+        statistics.put("Cameras#Content Status", response);
+    }
+
+    /**
+     * Retrieve device conferencing capabilities
+     *
+     * @param statistics map to put values to
+     * @throws Exception during http communication
+     */
+    public void getConferencingCapabilities(Map<String, String> statistics) throws Exception {
+        JsonNode response = doGet(buildHttpUrl(CONFERENCING_CAPABILITIES), JsonNode.class);
+        statistics.put("Conferencing Capabilities#Blast Dial", response.get("canBlastDial").asBoolean() ? "Available" : "Not Available");
+        statistics.put("Conferencing Capabilities#Audio Call", response.get("canMakeAudioCall").asBoolean() ? "Available" : "Not Available");
+        statistics.put("Conferencing Capabilities#Video Call", response.get("canMakeVideoCall").asBoolean() ? "Available" : "Not Available");
+    }
+
+    /**
+     * Retrieve current audio status
+     *
+     * @param statistics map to put values to
+     * @throws Exception during http communication
+     */
+    public void getAudioStatus(Map<String, String> statistics) throws Exception {
+        JsonNode response = doGet(buildHttpUrl(AUDIO), JsonNode.class);
+        statistics.put("Audio#Mute Locked", getJsonProperty(response, "muteLocked").asText());
+        statistics.put("Audio#Microphones Connected", getJsonProperty(response, "numOfMicsConnected").asText());
+    }
+
+    /**
+     * Add statistics property if the property exists
+     * Otherwise - skip
+     *
+     * @param statistics map to save property to
+     * @param name       name of the property
+     * @param node       to extract textual data from
+     */
+    private void addStatisticsProperty(Map<String, String> statistics, String name, JsonNode node) {
+        if (node != null && !node.isNull()) {
+            String value = node.asText();
+            if (!StringUtils.isNullOrEmpty(value.trim())) {

```

Comment:
StringUtils.isNullOrEmpty(value, **true**) will trim the string for you as well (Just FYI)...at least the StringUtils from symphony-commons project does.